react/features/chat/actions.native.ts

```diff
@@ -1,30 +1,33 @@
+import { IStore } from '../app/types';
 import { IParticipant } from '../base/participants/types';
 import { navigate } from '../mobile/navigation/components/conference/ConferenceNavigationContainerRef';
 import { screen } from '../mobile/navigation/routes';
 
 import { OPEN_CHAT } from './actionTypes';
+import { setFocusedTab } from './actions.any';
+import { ChatTabs } from './constants';
 
 export * from './actions.any';
 
 /**
- * Displays the chat panel.
+ * Displays the chat panel with the CHAT tab active.
  *
  * @param {Object} participant - The recipient for the private chat.
  * @param {boolean} disablePolls - Checks if polls are disabled.
  *
- * @returns {{
- *     participant: participant,
- *     type: OPEN_CHAT
- * }}
+ * @returns {Function}
  */
 export function openChat(participant?: IParticipant | undefined | Object, disablePolls?: boolean) {
-    if (disablePolls) {
-        navigate(screen.conference.chat);
-    }
-    navigate(screen.conference.chatandpolls.main);
+    return (dispatch: IStore['dispatch']) => {
+        if (disablePolls) {
+            navigate(screen.conference.chat);
+        }
+        navigate(screen.conference.chatandpolls.main);
```

Comment:
Shouldn't this be in the else loop? If polls are disabled, wouldn't the app still navigate to chatandpolls?